business-registry-dashboard/app/utils/reg-search.ts

```diff
@@ -13,8 +13,9 @@ export async function regSearch (queryStr: string): Promise<RegSearchResult[]> {
   const response = await $fetch<RegSearchResponse>(url, {
     headers: {
       Authorization: `Bearer ${token}`,
-      'x-apikey': config.registriesSearchApiKey,
-      'Account-Id': accountStore.currentAccount.id
+      'x-apikey': config.registriesSearchApiKey as string,
```

Comment:
`registriesSearchApiKey` was type "unknown" which causes a type error on `headers`. This fixes it.